wiki/Manual.md

```diff
@@ -2921,6 +2921,24 @@ To query for balance and get the amount of funds available for trading or funds
 fetchBalance (params = {})
 ```
 
+### Request
+  
+| Property | Required   | Description                   |
+|----------|------------|-------------------------------|
+| params   | No         | Params to override in request |
+
+### Response
+| Property      | Always | Description                                                                                   |
+|---------------|--------|-----------------------------------------------------------------------------------------------|
+| info          | Yes    | Response from exchange                                                                        |
+| timestamp     | No     | Time of request in ms                                                                         |
+| datetime      | No     | timestamp in ISO8601                                                                          |
+| free          | No     | balance free to use for each currenc. ``` { CURRENCY_CODE_1: 0, CURRENCY_CODE_2: 0, ... } ``` |
+| used          | No     | balance used in each currency  ``` { CURRENCY_CODE_1: 0, CURRENCY_CODE_2: 0, ... } ```        |
+| total         | Yes    | total balance per currency ``` { CURRENCY_CODE_1: 0, CURRENCY_CODE_2: 0, ... } ```            |
+| CURRENCY_CODE | No     | ``` { free: 0, used: 0, total: 0} ```                                                         |
+
```

Comment:
There should be a max width so that this doesn't wrap though, If you zoom out enough this doesn't wrap, and in vscode it doesn't wrap, but if this is excessively long it will wrap on most screens

Reply:
Let's use the proper markdown here. Apart from that it is transpiled to .rst to support ReadTheDocs (https://docs.ccxt.com/), so we have to satisfy both the markdown requirements and the rst requirements. Let's put exactly one whiteline above and below each header (starting with a hash sign `#`) and we should also replace the triple backtick blocks with a regular inline block (single ticks), otherwise it will break the rst.

Reply:
Tbh, on a second thought, using tables for listing the params is not the best way to represent this type of info, typographically. It's a list of params, so we should use lists for it to render properly on all screens and readers, including html, md, rst, or whatever. Something like this would work better, for example: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

Reply:
Also, we have the description of fetchBalance here and instead of duplicating it, we should simply add the crosslinks to the text: https://github.com/ccxt/ccxt/wiki/Manual#account-balance

Reply:
> Also, we have the description of fetchBalance here and instead of duplicating it, we should simply add the crosslinks and sub-headers to the text where necessary: https://github.com/ccxt/ccxt/wiki/Manual#account-balance

I edited that part

Reply:
Ah, then I don't think we need to duplicate the response structure as a table. And the request is also documented already.